src/components/speiling/sykepengsoknader/container/SykepengesoknaderSide.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import Side from "../../../../sider/Side";
+import Soknader from "../soknader/Soknader";
+import Brodsmuler from "../../Brodsmuler";
+import { SYKEPENGESOKNADER } from "../../../../enums/menypunkter";
+import Speilingvarsel from "../../Speilingvarsel";
+import Feilstripe from "../../../Feilstripe";
+import SideLaster from "../../../SideLaster";
+import { harForsoktHentetSoknader } from "../../../../utils/reducerUtils";
+import { useValgtPersonident } from "../../../../hooks/useValgtBruker";
+import { hentSoknader } from "../../../../data/sykepengesoknad/soknader_actions";
+
+const SykepengesoknaderSide = () => {
+  const dispatch = useDispatch();
+
+  const fnr = useValgtPersonident();
+
+  const { navbruker, soknader } = useSelector((state: any) => state);
```

Comment:
Går også an å bruke `const { navn } = useNavBrukerData()` her og da evt soknader-state i et eget kall.

Reply:
Ja, har sett det har blitt endel av den varianten, men hva er egentlig fordelen med å gjøre det slik i stedet for å bruke useSelector direkte når man skal  hente mange ting fra state samtidig?

Reply:
Jepp, kan endre.

Reply:
Endret.